### PR TITLE
Makes teslas absorb each other for power

### PR DESCRIPTION
**Pull request by ChesterTheCheesy**
# General Documentation
### Intent of your Pull Request
Alternative to [#11185].
When a tesla touches another tesla, it absorbs it, creating a more powerful tesla that can hold 2 more max miniballs, gets 1 tile longer zapping range, and gets more visibly blue itself per tesla consumed. The weaker tesla gets destroyed without any explosion, just a loud electric charging sound.

### Why is this change good for the game?
Removing tesla stacking because it's bad and lags the game

# Wiki Documentation
Can be noted i guess, as a trivia, since it can be theoritically used to make more power

### Briefly describe your PR and the impacts of it, in layman's terms. 
Makes energy balls (teslas) absorb each other for power, getting stronger and more dangerous when loose, but less dangerous and laggy whan multiple teslas.

### What should players be aware of when it comes to the changes your PR is implementing?
That it's more efficient to have multiple teslas each in its own containment rather than stack 50 teslas in one.

### What general grouping does this PR fall under? 
Engineering Tweak

# Changelog
:cl:  
tweak: teslas now absorb each other for more power, but very inefficiently
/:cl:

### CODE REVIEW

**Comment by patpol4** [comment]
better

**Comment by ChesterTheCheesy** [comment]
Yeah i think so too, it actually nerfs tesla stacking instead of removing it, but i'll leave both open and let the COUNCIL™ decide.

**Review by JamieD1 — APPROVED** [review]
